substra/cli/printers.py

```diff
@@ -312,8 +320,8 @@ class DatasetPrinter(AssetPrinter):
     download_message = 'Download this data manager\'s opener:'
 
 
-class TraintuplePrinter(AssetPrinter):
-    asset_name = 'traintuple'
+class BaseTraintuplePrinter(AssetPrinter):
+    asset_name = 'base_traintuple'
```

Comment:
It looks like this class isn't really used: there is only one sub-class, `TraintuplePrinter`, which overrides all props except `has_description`. You could refactor this so that BaseTraintuplePrinter is removed and TraintuplePrinter directly includes the `has_description = false` attribute.